burn-wgpu/src/ops/float_ops.rs

```diff
@@ -392,6 +393,12 @@ where
     }
 
     fn tanh<const D: usize>(tensor: FloatTensor<Self, D>) -> FloatTensor<Self, D> {
+        let tensor = mask_fill(
+            tensor.clone(),
+            greater_elem(tensor, F::from_elem(40.0)),
+            F::from_elem(40.0),
+        );
+
```

Comment:
Funny, I was about to work on a new [`clamp` operator](https://pytorch.org/docs/stable/generated/torch.clamp.html) that I needed to use for MobileNet. 

Do you mind adding a todo that instructs to replace with `clamp`?  And we need a note why we are doing it (basically the Metal has problems with over 40.0. BTW, why 40 if it fails at 43?

Reply:
Done and done. 
I had just put 40 because it's a more round number. Anyway tanh outputs 1.0 for everything above ~12. 
But 43 makes more sense